lang/src/org/partiql/lang/eval/physical/operators/RelationalOperatorKind.kt

```diff
@@ -14,6 +14,8 @@ enum class RelationalOperatorKind {
     UNPIVOT,
     FILTER,
     JOIN,
+    // TODO: Remove from experimental once https://github.com/partiql/partiql-docs/issues/31 is resolved and a RFC is approved
```

Comment:
Do we need this `TODO` here? it seems like there is no indication of experimental in `RelationalOperatorKind`